ui-admin/src/App.tsx

```diff
@@ -26,6 +26,16 @@ import AdminSidebar from './navbar/AdminSidebar'
 import NavContextProvider from 'navbar/NavContextProvider'
 import PopulateRouteSelect from './populate/PopulateRouteSelect'
 
+/** auto-scroll-to-top on any navigation */
+const ScrollToTop = () => {
+  const location = useLocation()
+  useEffect(() => {
+    // @ts-expect-error TS thinks "instant" isn't a valid scroll behavior.
+    window.scrollTo({ top: 0, left: 0, behavior: 'instant' })
```

Comment:
FWIW, `smooth` appears to be Bootstrap's influence as expressed in its `_reset.scss` (applied to the root `html` element). If we care to avoid the `@ts-expect-error`, we could override that opinion with our own of `auto` which will restore instant scrolling. Then we wouldn't have to specify `behavior` here at all.